bip-tap-universe.mediawiki

```diff
@@ -246,7 +246,7 @@ A Multiverse has the following structure:
 * Similar to normal Taproot Asset commitments, the Multiverse itself contains two nested MS-SMT trees. The upper tree commits to the set of asset groups observed, with the inner tree committing to the transaction history of each of the asset groups.
 * Upper tree structure:
 ** <code>key</code>: <code>asset_id</code> or <code>sha256(asset_key_family)</code>
-** <code>value</code>: <code>asset_group_tree_root || asset_group_sum</code>
+** <code>value</code>: <code>inner_tree_root</code>
```

Comment:
Maybe keep `asset_group_tree_root` as adds more context? Then can _just_ drop the sum value there in the key since that's implicit.